lib/src/err/mod.rs

```diff
@@ -770,6 +772,55 @@ pub enum Error {
 	/// The db is running without an available storage engine
 	#[error("The db is running without an available storage engine")]
 	MissingStorageEngine,
+
+	/// Error that can happen at any point during bootstrap
+	/// TODO change this to heartbeat? cleanup? cluster lifecycle?
```

Comment:
If this error variant is not stable yet, can we hide it from the docs for now? If it hits stable like this we won't be able to change it after the fact.

Reply:
Removed